MaxText/constants.py

```diff
@@ -0,0 +1,5 @@
+import os.path
+
+PKG_ROOT = os.path.abspath(os.path.dirname(__file__))
+
+__all__ = ["PKG_ROOT"]
```

Comment:
Why is this file and change needed ?

Reply:
This file can be removed as per my comment; just wanted a review of the rest before annoying your CI